verify-sna/README.md

```diff
@@ -1,64 +1,64 @@
-# verify-sna
-
-Sample Backend Silent Network Auth
-
-## Pre-requisites
-
-### Environment variables
-
-This project requires some environment variables to be set. A file named `.env` is used to store the values for those environment variables. To keep your tokens and secrets secure, make sure to not commit the `.env` file in git. When setting up the project with `twilio serverless:init ...` the Twilio CLI will create a `.gitignore` file that excludes `.env` from the version history.
-
-In your `.env` file, set the following values:
-
-| Variable | Description | Required |
-| :------- | :---------- | :------- |
-
-
-### Function Parameters
-
-`/blank` expects the following parameters:
-
-| Parameter | Description | Required |
-| :-------- | :---------- | :------- |
-
-
-`/hello-messaging` is protected and requires a valid Twilio signature as well as the following parameters:
-
-| Parameter | Description | Required |
-| :-------- | :---------- | :------- |
-
-
-## Create a new project with the template
-
-1. Install the [Twilio CLI](https://www.twilio.com/docs/twilio-cli/quickstart#install-twilio-cli)
-2. Install the [serverless toolkit](https://www.twilio.com/docs/labs/serverless-toolkit/getting-started)
-
-```shell
-twilio plugins:install @twilio-labs/plugin-serverless
-```
-
-3. Initiate a new project
-
-```
-twilio serverless:init example --template=verify-sna && cd example
-```
-
-4. Start the server with the [Twilio CLI](https://www.twilio.com/docs/twilio-cli/quickstart):
-
-```
-twilio serverless:start
-```
-
-5. Open the web page at https://localhost:3000/index.html and enter your phone number to test
-
-ℹ️ Check the developer console and terminal for any errors, make sure you've set your environment variables.
-
-## Deploying
-
-Deploy your functions and assets with either of the following commands. Note: you must run these commands from inside your project folder. [More details in the docs.](https://www.twilio.com/docs/labs/serverless-toolkit)
-
-With the [Twilio CLI](https://www.twilio.com/docs/twilio-cli/quickstart):
-
-```
-twilio serverless:deploy
-```
+# verify-sna

```

Comment:
I think we should use this readme to place a different content, something like we have [here](https://github.com/elkinnarvaez/function-templates/tree/main/verify-push-backend). Is this something we plan to make later?